coala_quickstart/generation/Bears.py

```diff
@@ -0,0 +1,104 @@
+from pyprint.NullPrinter import NullPrinter
+
+from coala_quickstart.Strings import bear_help
+from coala_utils.Question import ask_question
+from coalib.settings.ConfigurationGathering import load_configuration
+from coalib.misc.DictUtilities import inverse_dicts
+from coalib.collecting.Collectors import (
+    collect_all_bears_from_sections, filter_section_bears_by_languages)
+from coalib.output.printers.LogPrinter import LogPrinter
+
+
+def get_bears():
+    """
+    Get a dict of bears with the bear class as key.
+
+    :return:
+        A dict with bear classes as key and the list of sections
+        as value.
+    """
+    log_printer = LogPrinter(NullPrinter())
+    sections = load_configuration(None, log_printer)[0]
+    local_bears, global_bears = collect_all_bears_from_sections(
+        sections, log_printer)
+    return inverse_dicts(local_bears, global_bears)
+
+
+def filter_relevant_bears(bears, used_langauges):
+    """
+    From the bear dict, filter the bears per relevant language.
+
+    :param bears:
+        A dict of bears classes as key and the list of sections
+        as value.
+    :param used_langauges:
+        A list of tuples with language name as the first element
+        and percentage usage as the second element; sorted by
+        percentage usage.
+    :return:
+        A dict with language name as key and bear classes as value.
+    """
+    lang_map = {"all": "All"}
+    relevant_bears = {"All": []}
+    for lang in used_langauges:
+        if lang[0] == "Unknown":
+            continue
+        lang_map[lang[0].lower()] = lang[0]
+        relevant_bears[lang[0]] = []
+
+    for bear in bears:
+        bear_languages = [str(lang.lower()) for lang in bear.LANGUAGES]
+        for bear_lang in bear_languages:
+            if bear_lang in lang_map and lang_map[bear_lang] in relevant_bears:
+                relevant_bears[lang_map[bear_lang]].append(bear)
+    return relevant_bears
```

Comment:
this function as well should be in coala and we definitely have something *like* it, can we refactor this to use what we have in coala or improve the functions in coala so they can be reused here?